recipes/protobuf3-to-dict/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "protobuf3-to-dict" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 1e42c25b5afb5868e3a9b1962811077e492c17557f9c66f0fe40d821375d2b5a
+
+build:
+  skip: True  # [py2k]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - protobuf >=2.3.0
+    - python
+    - six
+    - pytest
+    - nosexcover
+  run:
+    - protobuf >=2.3.0
+    - python
+    - six
+
+test:
+  imports:
+    - protobuf_to_dict
+
+about:
+  home: "https://github.com/benhodgson/protobuf-to-dict"
+  license: Public Domain
+  license_family: PUBLIC-DOMAIN
+  license_file:
+  summary: "A teeny Python library for creating Python dicts from protocol buffers and the reverse. Useful as an intermediate step before serialisation (e.g. to JSON)."
+  doc_url:
+  dev_url:
```

Comment:
You can either fill these optional fields os remove them. Don't leave them blank.